src/virtual_dom/mod.rs

```diff
@@ -136,6 +136,12 @@ impl<T: AsRef<str>> From<Vec<T>> for Classes {
     }
 }
 
+impl PartialEq for Classes {
+    fn eq(&self, other: &Self) -> bool {
+        self.set.len() == other.set.len() && self.set.iter().eq(other.set.iter())
```

Comment:
Nice, did your tests catch this? Pretty odd that `IndexSet`'s equality implementation isn't transitive

Reply:
My tests did not catch this, but the vtag implementation did not use the `PartialEq` trait of `Classes`.
Instead it compared the inner classes.set, but since `diff_classes(...)` also needs to check if the classes changed, i moved the ordered equal check to the `PartialEq` trait of `Classes`.